<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node pkg="tj2_target" type="color_sensor.py" name="color_sensor" output="screen" respawn="true">
        <rosparam command="load" file="$(find tj2_target)/config/color_config.yaml" />
    </node>
</launch>
